spec/integration/build-swift/after/Example/Example.xcodeproj.yaml

```diff
@@ -0,0 +1,168 @@
+
+File References:
```

Comment:
Wow, how is this yaml file generated? Is there an option to convert `xcodeproj` into yaml format?